core/src/test/bridge_circuit_test_data.rs

```diff
@@ -0,0 +1,582 @@
+use super::common::citrea::get_bridge_params;
```

Comment:
add some module docs so that it's clear these are not tests but are just to generate the test_data and that they must be rerun when ... (idk when, probably whenever the circuit method id changes or there is some deposit breaking change)